src/screensComponents/modal/index.js

```diff
@@ -39,17 +39,21 @@ export default class Modal extends BaseComponent {
   renderTouchableOverlay() {
     const {overlayBackgroundColor, onBackgroundPress, accessibilityLabel = 'Dismiss'} = this.props;
     if (_.isFunction(onBackgroundPress) || !!overlayBackgroundColor) {
+      const isScreenReaderEnabled = Constants.accessibility.isScreenReaderEnabled;
+      const Container = isScreenReaderEnabled ? SafeAreaView : View;
+      const containerStyle = isScreenReaderEnabled
```

Comment:
You can tidy this a little by doing
`<View style={!isScreenReaderEnabled && [styles.touchableOverlay, {backgroundColor: overlayBackgroundColor}]} ...`

Reply:
Done